runtime/lua/vim/lsp/rpc.lua

```diff
@@ -176,11 +187,17 @@ function M.format_rpc_error(err)
   return table.concat(message_parts, ' ')
 end
 
+---@class vim.lsp.rpc.RpcError
+---@field code integer RPC error code defined by JSON RPC
```

Comment:
```suggestion
---@field code integer RPC error code defined by JSON RPC, see `vim.lsp.protocol.ErrorCodes`
```

Reply:
I had a similar comment earlier, but I deleted it since there is no help tag for `vim.lsp.protocol.ErrorCodes`. Is it ok to include it?

Reply:
Right, we don't have such a helptag (`|...|`), but as a reference to lua expression would be fine.